src/include/parser/parser_defs.h

```diff
@@ -60,8 +60,9 @@ enum class IndexType {
   INVALID = INVALID_TYPE_ID,  // invalid index type
   BWTREE = 1,                 // bwtree
   HASH = 2,                   // hash
-  SKIPLIST = 3,               // skiplist
-  ART = 4,                    // ART
+  // TODO(Matt) are we ever gonna support these again? if not we should remove them
```

Comment:
We could just go ahead and remove them. If/when someone adds them, they can easily add it here.